tests/test_lca.py

```diff
@@ -1527,6 +1527,32 @@ def test_single_summarize_to_output():
         assert '200,Bacteria,Proteobacteria,Gammaproteobacteria' in outdata
 
 
+
+def test_single_summarize_to_output_check_filename():
+    with utils.TempDirectory() as location:
+        db1 = utils.get_test_data('lca/delmont-1.lca.json')
+        input_sig = utils.get_test_data('lca/TARA_ASE_MAG_00031.sig')
+        in_dir = os.path.join(location, 'sigs')
+        os.mkdir(in_dir)
+        shutil.copyfile(input_sig, os.path.join(in_dir, 'q.sig'))
+
+        cmd = ['lca', 'summarize', '--db', db1, '--query', os.path.join(in_dir, 'q.sig'),
```

Comment:
side note, the use of `in_dir` here should render the use of `in_location` unnecessary in `utils.runscript(...)` - not a big deal, just mentioning it!

Reply:
ah, i see! i kept getting path errors from the lines below so i switched to setting the path with `os.path.join`. how would you write the code below using the `in_location` approach?

```
cmd = ['lca', 'summarize', '--db', db1, '--query', 'q.sig',
               '-o', os.path.join(location, 'output.txt')]
status, out, err = utils.runscript('sourmash', cmd, in_location=location)
```

Reply:
I think the disconnect is that `in_dir` is actually f"{location}/sigs", so you would need to specify `q.sig` as `sigs/q.sig` if you are using `utils.runscript(..., in_location=location)`.

Reply:
got it, thanks :)